lib/DirectAdmin_API.php

```diff
@@ -84,6 +84,14 @@ public function getDomainsMxRecords($domain, $username){
 			parse_str(substr($line, strlen($recordType)+1), $details);
 			foreach ($details as $record => $v){
 				$fixedRecord = str_replace('_', '.', $record);
+
+				// If the checkbox at CMD_API_DNS_MX isn't checked make sure we run an additional test
+				if ($isRemote == FALSE) {
+					if (substr($fixedRecord, -1) == '.' && strpos($fixedRecord, $domain) === false) {
+						$isRemote = TRUE;
```

Comment:
__[code_sniffer]__
* Spaces must be used for alignment; tabs are not allowed
* TRUE, FALSE and NULL must be lowercase; expected "true" but found "TRUE"

<!-- Comment by [SideCI](https://sideci.com) -->